src/client-apps/nuxeo-drive/nuxeo-drive-release-notes.md

```diff
@@ -9,81 +9,122 @@ review:
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2622](https://jira.nuxeo.com/browse/NXDRIVE-2622).
 
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2163](https://jira.nuxeo.com/browse/NXDRIVE-2163).
+### Improvements
 
+#### Fresh Account Tab
 
-### Fixed Conflicted Documents With Non-Standard Digest
+The Accounts tab in the Settings screen has been improved and is now visually closer to the interface of Direct Transfer and the new Synchronization tab.
 
-In Nuxeo Drive [5.0.0]({{page page='5.0.0-nuxeo-drive-release-notes'}}#handle-documents-with-non-standard-digest) we improved the handling of documents having non-standard digests. At the time we did not take care of conflicts for such documents, this is now done.
+SCREENSHOT
 
-The conflict resolution will be delayed until a valid digest will be sent by the server.
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2618](https://jira.nuxeo.com/browse/NXDRIVE-2618).
 
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2587](https://jira.nuxeo.com/browse/NXDRIVE-2587).
+#### About Features Sates
+
+In order to follow the [wygiwys principle](https://en.wikipedia.org/wiki/WYSIWYG), when manually changing the state of a feature, the state of all features will now be written into the local configuration file. the old behavior was saving only the impacted feature.
+
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2612](https://jira.nuxeo.com/browse/NXDRIVE-2612).
+
+## Direct Edit
+
+### Fixes
+
+The Direct Edit experience has been made more robust by better handling HTTP 404 (Not Found), 502 (Bad Gateway), 503 (Service Unavailable) and 504 (Gateway Timeout) errors.
+
+Related ticket: [NXDRIVE-2621](https://jira.nuxeo.com/browse/NXDRIVE-2621).
+
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2608](https://jira.nuxeo.com/browse/NXDRIVE-2608).
+
+## Direct Transfer
+
+### Features
+
+Allowing transfers into the Personal Space folder can be problematic. So we added a new option to hide it from the Direct Transfer window: [dt-hide-personal-space](https://doc.nuxeo.com/client-apps/nuxeo-drive/#dt-hide-personal-space).
+
+The option can be set globally from the server configuration, and it will be taken into account almost instantly by all clients without requiring a restart of the application.
+
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2632](https://jira.nuxeo.com/browse/NXDRIVE-2632).
+
+### Fixes
+
+When choosing a folder or a file in the direct Transfer window, symbolic links are now ignored to prevent dangerous or unhandled behaviors.
```

Comment:
```suggestion
When choosing a folder or a file in the Direct Transfer window, symbolic links are now ignored to prevent dangerous or unhandled behaviors.
```